src/KeyedFrames.jl

```diff
@@ -3,7 +3,7 @@ module KeyedFrames
 using DataFrames
 
 import DataFrames: deletecols!, deleterows!, first, index, last, ncol, nonunique, nrow,
-    permutecols!, rename, rename!, SubDataFrame, unique!
+    permutecols!, rename, rename!, SubDataFrame, unique!, DataFrameRow, select!, _check_consistency
```

Comment:
Let's try to follow BlueStyle here. I would separate the types entirely here since this line is already so long.
```julia
using DataFrames: DataFrameRow, SubDataFrame
using DataFrames: _check_consistency, deletecols!, deleterows!, first, index, last, ncol, nonunique, nrow,
    permutecols!, rename, rename!, select!, unique!
```